examples/proc_shims.py

```diff
@@ -0,0 +1,113 @@
+"""
+This script takes in the datasets acquired with run_shimming in spincore_apps.
+After properly phasing, the signal energy for each voltage and current is
+calculated to print the final optimal voltage setting for the tested channel.
+"""
+from pylab import r_, exp, subplot
+from pyspecdata import figlist_var, find_file
+
+with figlist_var() as fl:
+    for filename, nodename, color_tuple, label_str in [
+        ("221213_ShimOn3", "shims_addr5_Z_yOpt", ("teal", "pink"), 
+            ("opt y (run 4)", "opt y + opt z1 (run 4)")),
+    ]:
+        s = find_file(filename + ".h5", exp_type="ODNP_NMR_comp/echoes", expno=nodename)
+        s = s.C.mean("nScans")
+        tau = s.get_prop("acq_params")["tau_us"] - 10.0
+        if "t2" not in s.dimlabels:
+            s.set_units("t", "s")
+            s.chunk("t", ["ph1", "ph2", "t2"], [4, 2, -1])
+            s.setaxis("ph1", r_[0, 1, 2, 3] / 4)
+            s.setaxis("ph2", r_[0, 2] / 4)
```

Comment:
newest PR for the acquisition script does this but to process older data this is needed